GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -52,31 +52,35 @@ BOOL GDTCORReachabilityFlagsContainWWAN(SCNetworkReachabilityFlags flags) {
 
 GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
 #if TARGET_OS_IOS
-  static NSDictionary<NSString *, NSNumber *> *ctRadioAccessTechnologyToNetworkSubTypeMessage;
+  static NSDictionary<NSString *, NSNumber *> *CTRadioAccessTechnologyToNetworkSubTypeMessage;
   static CTTelephonyNetworkInfo *networkInfo;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
```

Comment:
I think this is doable because all the members of the dictionary compile-time constants. I think you can probably discard the dispatch_once and just do
```
   static NSDictionary<NSString *, NSNumber *> *CTRadioAccessTechnologyToNetworkSubTypeMessage = @{
 <all the stuff here>
    };	   
 };
```

If that doesn't work you can leave as-is

Reply:
It gets a error "initializer element is not a compile-time constant", I will leave as-is.

Reply:
SGTM